service/league_service/league_service.py

```diff
@@ -210,7 +210,21 @@ async def _persist_score(
                     game_count=new_score.game_count,
                 )
             )
-            await conn.execute(upsert_sql)
+            await conn.execute(score_insert_sql)
```

Comment:
Technically, the point of a journal is that you write to it before writing the actual value. Then it can be used to recover from errors if writing the value fails, although we're not really doing that so in practice it doesn't matter. Still, since it doesn't matter which goes first, I think you should put the journal insert first.

Reply:
Good idea. I basically just copied how it was implemented in the server.

Reply:
Lobby server writes the journal first